metrics/testnet-monitor.json

```diff
@@ -502,7 +502,7 @@
           "hide": false,
           "orderByTime": "ASC",
           "policy": "default",
-          "query": "SELECT ROUND(MEAN(\"sum\")) FROM ( SELECT sum(\"count\") FROM \"$testnet\".\"autogen\".\"counter-banking_stage-process_transactions\" WHERE $timeFilter GROUP BY time(1s) )\n\n",
+          "query": "SELECT ROUND(MEAN(\"sum\")) FROM ( SELECT sum(\"count\") FROM \"$testnet\".\"autogen\".\"counter-banking_stage-record_transactions\" WHERE $timeFilter GROUP BY time(1s) )\n\n",
```

Comment:
A similar change is needed in mean, max and total transactions counts in the dashboard

![image](https://user-images.githubusercontent.com/40076733/57552679-842a7200-7321-11e9-913b-2c28d4560601.png)